src/meta/processors/zone/DropZoneProcessor.cpp

```diff
@@ -24,43 +24,12 @@ void DropZoneProcessor::process(const cpp2::DropZoneReq& req) {
     return;
   }
 
```

Comment:
Where do we check host is empty? The space properties is not impacted?

Reply:
`DROP ZONE` will be fixed in the next pr. This one only remove some group related function